src/snmalloc/mem/localalloc.h

```diff
@@ -212,39 +134,48 @@ namespace snmalloc
         }
 
         return capptr_chunk_is_alloc(capptr_to_user_address_control(chunk));
-      });
+      };
+
+      return CheckInit::check_init(
+        fast_path,
+        [](size_t size, LocalAllocator* self) {
+          return self->alloc_not_small<zero_mem, CheckInitDefault>(size);
+        },
+        size,
+        this);
```

Comment:
It's a bit of a pity that something so formulaic can't, AFAIK, be templated away.